spring-boot-project/spring-boot-tools/spring-boot-gradle-plugin/src/main/java/org/springframework/boot/gradle/tasks/bundling/BootArchiveSupport.java

```diff
@@ -150,11 +150,11 @@ CopyAction createCopyAction(Jar jar, ResolvedDependencies resolvedDependencies,
 	}
 
 	private Integer getDirMode(CopySpec copySpec) {
-		return getMode(copySpec, "getDirPermissions", () -> copySpec.getDirMode());
+		return getMode(copySpec, "getDirPermissions", copySpec::getDirMode);
 	}
 
 	private Integer getFileMode(CopySpec copySpec) {
-		return getMode(copySpec, "getFilePermissions", () -> copySpec.getFileMode());
+		return getMode(copySpec, "getFilePermissions", copySpec::getFileMode);
```

Comment:
We can't use method references here. See https://github.com/spring-projects/spring-boot/issues/41599.

Reply:
My bad, I missed that. Reverted the changes in 06f08d88c2feb2d2e44da2a76df77efc07a8a502.